theano/sandbox/gpuarray/conv_desc.c

```diff
@@ -0,0 +1,34 @@
+#section support_code_apply
+
+int conv_desc(PyArrayObject *filt_shp,
+              cudnnConvolutionDescriptor_t *desc) {
```

Comment:
It part of the support code, but is called by the c_code to perform the computation.